services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(tickets: list[Ticket],
+                 username: str,
+                 date: str = None) -> QuerySet:
+    user = User.objects.get(username=username)
+
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+        order.save(update_fields=["created_at"])
```

Comment:
The `save()` method is called twice for the `Order` object. According to the checklist, the `save()` method should be called only once if possible. You can set the `created_at` attribute when creating the `Order` object.